dictionaries/attributes.definition.json

```diff
@@ -53,6 +53,9 @@
 	"attribute_edupersonuniqueid": {
 		"en": "Person's non-reassignable, persistent pseudonymous ID at home organization"
 	},
+	"attribute_edupersonorcid": {
+		"en": "ORCID researcher identifiers belonging to the principal"
```

Comment:
This may be a nitpick, but maybe this should just be "ORCID researcher identifiers", to be more in line with (most) other descriptions in this file (e.g. "Telephone number", not "Telephone number belonging to the principal").